modules/node/src/channel/channel.service.ts

```diff
@@ -279,6 +279,24 @@ export class ChannelService {
     return state;
   }
 
+  async getChannelStateByMultiSig(multisigAddress: string): Promise<any> {
+    const channel = await this.channelRepository.findByMultisigAddress(multisigAddress);
+    if (!channel) {
```

Comment:
why not just return null?

Reply:
ease of debugging, mostly. I can change this if you want but otherwise it would just fail silently and null responses would need to be handled better upstream